core/Rename.cpp

```diff
@@ -356,4 +359,9 @@ namespace olympia
         }
     }
 
+
+    void Rename::onStartingTeardown_(){
+        sparta_assert(current_stall_ == StallReason::NOT_STALLED, "Rename is stalled when entering teardown, possible lockup in Rename?")
```

Comment:
@knute-sifive Was thinking either to have an assert like this to ensure that there is no ending stall when teardown begins, or should I do a notification similar to https://github.com/riscv-software-src/riscv-perf-model/pull/108

Reply:
Yeah, you can't have this assertion.  It's possible that Rename is stalled waiting for free registers AND the ROB was told to stop simulation on instruction _N_.  This is a valid case and you don't want to assert on that. 

In #108 (still under review at this time), having the ROB broadcast "I stopped simulation" and Rename listening for that and determining if it's a cause in this function is appropriate.

Reply:
I'll remove the teardown and assertion for now, and I'll add in the rename debugging once https://github.com/riscv-software-src/riscv-perf-model/pull/108 is merged.